.github/workflows/build.yml

```diff
@@ -17,7 +17,8 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                symfony: ["^5.4"]
+                symfony: ["^5.4", "^6.0"]
+                dbal: ["^2.0", "^3.0"]
```

Comment:
As we are testing these two values, it would be good to make a clear contract in our composer.json don't you think?